java/connector-node/risingwave-sink-iceberg/src/main/java/com/risingwave/connector/catalog/JniCatalogWrapper.java

```diff
@@ -62,6 +64,38 @@ public String updateTable(String updateTableRequest) throws Exception {
         return RESTObjectMapper.mapper().writer().writeValueAsString(resp);
     }
 
+    /**
+     * Create table through this prox.
+     *
+     * @param namespace Namespace.
+     * @param createTableRequest Request serialized using json.
+     * @return Response serialized using json.
+     * @throws Exception
+     */
+    public String createTable(String namespace, String createTableRequest) throws Exception {
+        Namespace ns = null;
+        if (namespace == null) {
+            ns = Namespace.empty();
+        } else {
+            ns = Namespace.of(namespace);
+        }
+        CreateTableRequest req =
```

Comment:
just a question:
Can we distinguish between these similar variables (createTableRequest, CreateTableRequest, req) in terms of naming?  

Use more explicit naming, even with types, e.g..
1. createTableRequestStr
2. reqEntity